### ISSUE TITLE
Prefix Auto Kick geht nicht

### ISSUE THREAD

**Issue by Skillkiller** [zekroTJA/KnechtBot2#4]
Hallo,
ich habe bei meinem System Bot bewusst den Prefix nicht regestiert um zu testen ob der Kick funktioniert. Leider hat dies nicht geklappt.

**Comment by zekroTJA** [comment]
Hey ^^

Könntest du das eventuell nochmal etwas genauer beschreiben?
Bin mir gerade nicht ganz sicher, was du meinst.

Du hast also in der Config den Prefix nicht eingestellt und wolltest dann den Kick-Command nutzen, richtig?

**Comment by Skillkiller** [comment]
Nein, ich habe meinen Userbot auf den Server geholt. Danach soll ich ja eigentlich den Prefix einstellen, denn wenn ich das nicht tue soll der Bot automatisch nach 24 wieder gekickt werden. Und der automatische Kick wurde aber nicht ausgeführt obwohl der Bot jetzt schon 2 Tage oder länger auf dem Server ist ohne regestrierten Prefix

**Comment by zekroTJA** [comment]
Achso okay, das meinst du. Okay, das werde ich mir mal bei Gelegenheit die Woche anschauen. Danke :)
